lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -143,14 +147,15 @@ def __init__(self, charm: Object):
         )
         self.framework.observe(self.charm.on.create_backup_action, self._on_create_backup_action)
         self.framework.observe(self.charm.on.list_backups_action, self._on_list_backups_action)
+        self.framework.observe(self.charm.on.restore_backup_action, self._on_restore_backup_action)
 
     @property
     def _plugin_status(self):
         return self.charm.plugin_manager.get_plugin_status(OpenSearchBackupPlugin)
 
     def _on_list_backups_action(self, event: ActionEvent) -> None:
         """Returns the list of available backups to the user."""
-        if not self._check_action_can_run(event):
+        if not self._can_unit_perform_backup(event):
             event.fail("Failed: backup service is not configured yet")
```

Comment:
still UX should be improved. e.g. id should be a time of backup start, it should be a table with in-progress/failed/successfull status, etc
```
$ juju run postgresql/3 list-backups
...
backups: |-
  backup-id             | backup-type  | backup-status
  ----------------------------------------------------
  2023-10-19T11:36:23Z  | physical     | finished
```